packages/backend/src/open_payments/payment/incoming/routes.ts

```diff
@@ -56,24 +58,18 @@ async function getIncomingPayment(
   )
   if (!incomingPayment) return ctx.throw(404)
 
-  const amountReceived = await deps.accountingService.getTotalReceived(
```

Comment:
`accountingService` can be removed from [`ServiceDependencies`](https://github.com/interledger/rafiki/blob/ae2a7aff5193b5fc5f53da331eedcecdb77ada6f/packages/backend/src/open_payments/payment/incoming/routes.ts#L20)